src/App.js

```diff
@@ -111,14 +120,11 @@ function App() {
         setShowAlert(!showAlert);
     }
 
-    if (loading) {
+    if (loadingTask || loadingSubtask) {
         return <div className={"loading"}>Loading Task List...</div>
     }
```

Comment:
No check for error. This can cause taskList to be undefined and then Tasks may crash